packages/features/orgs/SubdomainProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ReactNode } from "react";
+import { createContext, useContext, useEffect, useState } from "react";
+
+import { trpc } from "@calcom/trpc";
+
+interface SubdomainContextType {
+  subdomain: string | undefined;
+  isSubdomain: boolean;
+  logoSrc?: string;
+}
+
+const SubdomainContext = createContext<SubdomainContextType>({ subdomain: undefined, isSubdomain: false });
+
+export const useSubdomainContext = () => {
+  return useContext(SubdomainContext);
+};
+
+interface SubdomainProviderProps {
+  children: ReactNode;
+}
+
+export const SubdomainProvider = ({ children }: SubdomainProviderProps) => {
+  const [enabled, setEnabled] = useState(true);
+  const { data } = trpc.viewer.orgs.getLogo.useQuery(undefined, { enabled });
```

Comment:
We can get the same behavior by just doing this. No need to use useState here.

```suggestion
  const { data } = trpc.viewer.orgs.getLogo.useQuery(undefined, {
    staleTime: Infinity,
  });
```